.circleci/config.yml

```diff
@@ -256,6 +256,9 @@ jobs:
             else
               ./scripts/build_review_app.sh $review_app_name
             fi
+      - run:
+          name: Create Cloudflare DNS namespace
+          command: node scripts/createReviewAppNamespace.js
```

Comment:
* "Namespace" is a different kubernetes thing above. Can this just be called `createReviewAppSubdomain` or `...DNS`?
* Also, the review app label is extracted above and passed as an arg to the other scripts. Maybe this could just be another line in the above script, and reuse that label rather than repeat its logic.

Reply:
Updated the script name 👍 

Re extracting subdomain from branch name, I moved this into a separate job since hokusai/beta doesn't provide a clean way to access `node` (for something like this) so its still duplication. This is likely to go away shortly anyways, so will punt on that for now.